server/src/main/java/org/elasticsearch/search/retriever/RankDocsRetrieverBuilder.java

```diff
@@ -93,7 +93,8 @@ public QueryBuilder explainQuery() {
         var explainQuery = new RankDocsQueryBuilder(
             rankDocs.get(),
             sources.stream().map(RetrieverBuilder::explainQuery).toArray(QueryBuilder[]::new),
-            true
+            true,
+            Float.MIN_VALUE
```

Comment:
This should use a default member that you create and reference as suggested earlier. Alternately consider a convenience constructor that sets the default.